lib/states/create_account.rb

```diff
@@ -20,7 +20,7 @@ def next
 
     private
 
-    def errors?
+    def errors
```

Comment:
again. Method with "?" should return a boolean value. Nothing more. Return '?' and move puts logic in different place